yarn-project/aztec-faucet/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "@aztec/aztec-faucet",
```

Comment:
nitpicky but I think we should name this just `faucet` rather than `aztec-faucet`